x-pack/plugins/security_solution/scripts/endpoint/endpoint_agent_runner/elastic_endpoint.ts

```diff
@@ -50,15 +51,17 @@ export const enrollEndpointHost = async (): Promise<string | undefined> => {
     log,
     kbnClient,
     options: { version, policy },
+    settings,
   } = getRuntimeServices();
 
   log.info(`Creating VM and enrolling Elastic Agent`);
   log.indent(4);
 
   try {
-    const uniqueId = Math.random().toString(32).substring(2).substring(0, 4);
+    const uniqueId = Math.random().toString().substring(2, 6);
```

Comment:
converted to just appending a random 4-digit number - easier to use when trying to delete them.